src/actions/onboardingActions.js

```diff
@@ -311,8 +312,10 @@ export const finishOnboardingAction = (retry?: boolean, recoveryData?: Object) =
 
     await dispatch(getTutorialDataAction());
 
+    const feautureOnboarding = firebaseRemoteConfig.getString(REMOTE_CONFIG.FEATURE_ONBOARDING);
```

Comment:
```suggestion
    const enableOnboarding = firebaseRemoteConfig.getString(REMOTE_CONFIG.FEATURE_ONBOARDING);
```

Reply:
@mdjastrzebski Thank you for reviewing.The changes have been done.